attach-remotes

```diff
@@ -10,7 +10,14 @@ Jenkins.instance.nodes.each {
 
 
 
-String[] agentList = ["ci-james-01", "ci-james-02", "ci-james-03", "ci-james-04", "james-charge-01", "james-charge-02"]
+String[] agentList = ["ci-james-01",
+    "ci-james-02",
+    "ci-james-03",
+    "ci-james-04",
+    "ci-james-06",
```

Comment:
where is ci-james-05 ?

Reply:
This machine used for another purpose (Jenkins 2 development).